### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by mdr0id**
Update naming to remove buildbot from PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,4 +2,4 @@ Please ensure this checklist is followed for any pull requests for this repo. Th
 * [ ] Relevant documentation for this PR has to be completed and reviewed by @mdr0id before the PR can be merged
```
Comment (nuttycom) [inline .github/pull_request_template.md:2]: This is something that we have never enforced, and honestly it doesn't seem like a good idea to bottleneck every PR on one individual. Instead, I think that we should have a requirement that the changelog has been updated.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -2,4 +2,4 @@ Please ensure this checklist is followed for any pull requests for this repo. Th
...
 * [ ] A test plan for the PR must be documented in the PR notes and included in the test plan for the next regular release
```
Comment (nuttycom) [inline .github/pull_request_template.md:3]: We need some documentation about how these test plans are intended to be written - I don't know of even any examples - and it's not clear how to include it in the test plan.

**Review by nuttycom — COMMENTED** [review]
If we're going to update the PR template, we should really do a bit more of a comprehensive rework, since we never actually use or follow it as it's currently written.

**Comment by daira** [comment]
Not needed for 5.1.0. This should be merged after we have fixed the CI bottleneck.

**Comment by str4d** [comment]
Superceded by #6222.

**Pull request closed without merging**